### PR TITLE
Add badges in README.md

### PR DESCRIPTION
**Pull request by baumstern**

### CODE REVIEW

**Review by CPerezz — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # A collection of Elliptic Curves for ZkCrypto traits
 
+[![crates.io version](https://img.shields.io/crates/v/halo2curves.svg)](https://crates.io/crates/halo2curves)
+[![docs.rs availability](https://img.shields.io/docsrs/halo2curves?label=docs.rs)](https://docs.rs/halo2curves/)
+[![Build status](https://img.shields.io/github/actions/workflow/status/privacy-scaling-explorations/halo2curves/ci.yml?branch=main)](https://github.com/privacy-scaling-explorations/halo2curves//actions)
```
Comment (han0110) [inline README.md:5]: Just a nitpick that could be ignored (the link is still working with double slash)

```suggestion
[![Build status](https://img.shields.io/github/actions/workflow/status/privacy-scaling-explorations/halo2curves/ci.yml?branch=main)](https://github.com/privacy-scaling-explorations/halo2curves/actions)
```

**Review by han0110 — APPROVED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (baumstern) [reply README.md]: Reflected in 428ac24

**Pull request merged**